pkg/k8s/client.go

```diff
@@ -750,6 +756,27 @@ func (c *Client) CreateOrUpdateNamespace(name string, labels, annotations map[st
 	return nil
 }
 
+// ForceDeleteNamespace allows a namespace to be forcibly deleted
+// by overriding it's finalizers
+func (c *Client) ForceDeleteNamespace(ns string, timeout time.Duration) error {
+	k8s, err := c.GetClientset()
+	if err != nil {
+		return fmt.Errorf("ForceDeleteNamespace: failed to get client set: %v", err)
+	}
+
+	namespace, err := k8s.CoreV1().Namespaces().Get(ns, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("ForceDeleteNamespace: failed to get namespace: %v", err)
+	}
+	namespace.Spec.Finalizers = []v1.FinalizerName{}
+	_namespace, err := k8s.CoreV1().Namespaces().Finalize(namespace)
+	if err != nil {
+		return fmt.Errorf("ForceDeleteNamespace: error removing finalisers: %v", err)
+	}
+	c.Infof("%v", _namespace)
```

Comment:
I think this should be a warning at the top of method

Reply:
yes, agreed.
Moved in 68a03d04271d5892874251f31ce0c7186ace7713